internal/Picker/Picker.js

```diff
@@ -938,110 +957,113 @@ const PickerBase = class extends ReactComponent {
 		}
 
 		return (
-			<Layout
-				{...voiceProps}
-				{...rest}
-				align="center space-around"
-				aria-controls={joined ? id : null}
-				aria-disabled={disabled}
-				aria-label={this.calcAriaLabel(valueText)}
-				className={className}
-				component={Component}
-				data-webos-voice-intent="Select"
-				data-webos-voice-labels-ext={voiceLabelsExt}
-				disabled={disabled}
-				holdConfig={holdConfig}
-				inline
-				onBlur={this.handleBlur}
-				onDown={this.handleDown}
-				onFocus={this.handleFocus}
-				onHold={this.handleHold}
-				onKeyDown={this.handleKeyDown}
-				onKeyUp={this.handleKeyUp}
-				onUp={this.handleUp}
-				onMouseDown={this.handleMouseDown}
-				onMouseLeave={this.clearPressedState}
-				orientation={orientation}
-				ref={this.initContainerRef}
-				{...spottablePickerProps}
-			>
-				{isHorizontalJoined ?
-					null :
-					<Cell
-						{...voiceProps}
-						align={joined ? 'stretch' : null}
-						aria-controls={!joined ? incrementerAriaControls : null}
-						aria-label={this.calcIncrementLabel(valueText)}
-						className={css.incrementer}
-						component={PickerButton}
-						data-webos-voice-label={joined ? this.calcButtonLabel(!reverse, valueText) : null}
-						disabled={incrementerDisabled}
-						holdConfig={holdConfig}
-						icon={incrementIcon}
-						joined={joined}
-						onDown={this.handleIncrement}
-						onHold={this.handleIncrement}
-						onKeyDown={this.handleIncKeyDown}
-						onSpotlightDisappear={onSpotlightDisappear}
-						shrink
-						spotlightDisabled={spotlightDisabled}
-					/>
-				}
-				<Cell
+			<>
+				{title ? <Heading className={classnames(componentCss.title, {[componentCss.inline]: inlineTitle})} size="tiny">{title}</Heading> : null}
```

Comment:
I added this part and I didn't change belows.

Reply:
I'd rather put this somewhere else not here since this component represents a single picker which is also used in Date or Time picker.
Let's move this to Picker and RangePicker. If you don't like to write the same code twice, make an internal hoc or hook.